pr_watch/models.py

```diff
@@ -72,6 +73,37 @@ class WatchedFork(models.Model):
         db_index=True,
         help_text='Github fork name that will be watched for PRs. E.g.: open-craft/edx-platform'
     )
+    ansible_appserver_repo_url = models.URLField(
+        max_length=256,
+        blank=True,
+        null=True,
+        help_text=('The repository to pull the default Ansible playbook from.')
```

Comment:
@gabor-boros not entirely sure about this, but it looks pretty fragile to allow this to be blank and null, I guess this is done so it doesn't affect previous instances.
I think we can create instances from django admin, this means that these new fields are not necessary when doing that? And if that's the case, why were those fields added on the tests

Reply:
@eLRuLL I'm not entirely sure I got your question right, but these fields are for "overriding" the default configuration of instances. Hence it is required to allow blank and null values (otherwise we couldn't use default values at all

Reply:
yeah no problem, I think I was getting confused with instance configuration, but this is just for watching PRs